capital_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './base_decorator'
+
+class CapitalDecorator < Base_Decorator
+  def correct_name
+    @nameable.correct_name.upcase
+  end
```

Comment:
* Kindly fix this error I get when I try to decorate a person 
* Also ensure after fixing the error Try the following code and check if you managed to decorate your person

> person = Person.new(22, 'maximilianus')
 

>  person.correct_name

 

>  capitalizedPerson = CapitalizeDecorator.new(person)

 

>  capitalizedPerson.correct_name

>   capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)

>   capitalizedTrimmedPerson.correct_name

![image](https://user-images.githubusercontent.com/52497246/200765347-8cd7b190-46a8-4097-9169-02ccf505a939.png)